mobx/lib/mobx.dart

```diff
@@ -61,6 +61,7 @@ export 'package:mobx/src/core.dart'
         NotificationHandlers,
         ReactionImpl,
         DerivationState;
+export 'package:mobx/src/api/extensions.dart';
 
 /// The current version as per `pubspec.yaml`
 const version = '0.3.10';
```

Comment:
We will need to bump the version in `pubspec.yaml`

Reply:
done